src/components/main_screens/HomeScreen.js

```diff
@@ -127,8 +136,8 @@ export default function HomeScreen({ navigation }) {
             getGraphData={getGraphData}
             setSelectedTimeframe={setSelectedTimeframe}
             selectedTimeframe={selectedTimeframe}
-            handlePressInTouchableElement={handlePressInTouchableElement}
```

Comment:
We need this so that the user can press on the graph to see specific data points without it scrolling the screen down